packages/foreman/rubygem-sidekiq/rubygem-sidekiq.spec

```diff
@@ -17,12 +17,13 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 2.2.2
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(redis) >= 3.3.5
-Requires: %{?scl_prefix}rubygem(redis) < 5
+Requires: %{?scl_prefix}rubygem(redis) >= 4.5
+Requires: %{?scl_prefix}rubygem(redis) < 4.6.0
```

Comment:
it's cool that this was bumped, but then not packaged? ;-)

Reply:
https://github.com/theforeman/foreman-packaging/pull/7734

Reply:
I was afraid of that, but didn't have time to check when I submitted the PR. I had hoped repoclosure would catch it.